### DISCUSSION TITLE
Roboshop-terraform Jenkins build error

### DISCUSSION THREAD

**Discussion by srinivasdharni** [learndevopsonline/learndevopsonline#1626] [Q&A] [answered]
Hi Raghu,
when running the  jenkins build for roboshop-terraform project.getting the below build error 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/51589511/dc4d8322-b25b-4847-ba7a-797c18958b31)

**Comment by sreedharm07** [comment, marked as answer]
hi @srinivasdharni 
i says minimum vpc limit reached 
raise ticket to aws to increase vpc limit
